app/components/SearchTableLayout.tsx

```diff
@@ -22,6 +22,17 @@ export const SearchTableLayoutComponent: React.FunctionComponent<Props> = (
     isLoading
   } = props;
 
+  const noSearchResults =
+    body.props.children.length === 0 ? (
+      <span id="no-search-results">
+        <Alert variant="secondary">
+          <p>&quot;No matching results to show.&quot;</p>
+          <p>Clicking &apos;Reset&apos; will clear your search parameters.</p>
```

Comment:
Did we get a design done for this feature. I believe one of the recommendations from @rajpersram, to be implemented in the future, was to remove the `Reset` button: https://youtrack.button.is/issue/GGIRCS-1626

Reply:
No, there was no specific design done for this. There was just a suggestion of what to show in the card, I added the 'Reset button' instruction and it could definitely come out.